server/jamovi/server/instance.py

```diff
@@ -270,13 +271,33 @@ def _on_save(self, request):
         path = request.filename
         path = Instance._normalise_path(path)
 
-        formatio.write(self._data, path)
+        try:
+            file_exists = os.path.isfile(path)
+            successfulWrite = False
+            if file_exists is False or request.override is True:
```

Comment:
i think you've used `overwrite` before, where as this is `override`. could/should these by made consistent?